### PR TITLE
DPTOPICS-1184/update docs to reflect shift from `dl` to `ul`

### PR DESCRIPTION
**Pull request by loki-bbc**
Ticket: https://jira.dev.bbc.co.uk/browse/DPTOPICS-1184
Slack thread (for more context): [#help-webcore-ux](https://bbc-dpg.slack.com/archives/CSGJQ1QFQ/p1651597827698769)

Following the changes made to to support Metadata Snippets without a title ([web PR](https://github.com/bbc/web/pull/8173)), this PR updates the associated GEL documentation with said component and respective changes.

These doc changes would eventually be visible at https://bbc.github.io/gel/components/metadata-strips/

### CODE REVIEW

**Review by JoshTumath — APPROVED** [review]
EDIT: Sorry, I thought I was reviewing a different PR. 😬

File: _content/components/metadata-strips.md
Comment (JoshTumath) [inline _content/components/metadata-strips.md]: There's no distinction between titles and definitions any more. Maybe say this:
```suggestion
The purpose of `nowrap` is to ensure items are never split across lines. This aids scanning and comprehension.
```

File: _content/components/metadata-strips.md
Comment (JoshTumath) [inline _content/components/metadata-strips.md]: Can you update this link to the WCAG 2.1 guidelines? https://www.w3.org/WAI/WCAG21/Understanding/use-of-color

**Review by JoshTumath — COMMENTED** [review]
OK, here's an actual review. 😅 I think it looks good. I've never reviewed a GEL tech doc change before, so you may want a second opinion.

**Review by sbeaumontweb — COMMENTED** [review]
Heyo, could we update the version on this component to 1.0.0? I'd call this significant changes to this piece 😄 Will merge once done, cheers!

**Comment by loki-bbc** [comment]
> Heyo, could we update the version on this component to 1.0.0? I'd call this significant changes to this piece 😄 Will merge once done, cheers!

No problems, thank you for the review have done that! As we're still awaiting an articles review on the web PR (https://github.com/bbc/web/pull/8173), and for our testers to comb through it, maybe we hold fire on merging this until we're ready to merge that first? I don't anticipate there being any changes needed that would need to be reflected in a doc change, but just a precautionary measure.

Edit: I tried adding a `DO NOT MERGE` label, but it seems I can't add labels to this PR? Maybe another permissions thing, but I would've thought a fork would let me. I've edited the title of the PR just now to reflect this. I'll ping someone once the other PR is ready to be merged.

**Pull request merged**
